wmfdata/utils.py

```diff
@@ -41,6 +42,19 @@ def pd_display_all(df):
     ):
         display(df)
 
+def suppress_t324135(fn):
```

Comment:
I would prefer a name that makes sense even if you don't look at the task or the docstring. How about `suppress_pandas_dbapi_warning`?

Reply:
Makes sense!